.github/workflows/ci.yml

```diff
@@ -4,40 +4,20 @@ on:
   - push
   - workflow_dispatch
 
-env:
-  MAVEN_FLAGS: "-B --no-transfer-progress"
-  MAVEN_OPTS: "-Xmx2G -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000 -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3"
-
 jobs:
-  tests:
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        java-version:
-          - 8
-        profile:
-          - travis
-          - integration-mysql
-          - integration-postgresql
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-      - name: Setup Java
-        uses: actions/setup-java@v1
-        with:
-          java-version: ${{ matrix.java-version }}
-      - name: Configure Sonatype mirror
-        uses: s4u/maven-settings-action@v2.3.0
-        # Go to Sonatype directly to avoid delay syncs (could get rid of this if actions/setup-java were to support mirrors).
-        with:
-          mirrors: '[{"id": "oss-releases", "name": "Sonatype releases", "mirrorOf": "*", "url": "https://oss.sonatype.org/content/repositories/releases/"}]'
-      - name: Tests
-        env:
-          MAVEN_PROFILE: ${{ matrix.profile }}
-          BRAINTREE_ENVIRONMENT: sandbox
-          BRAINTREE_MERCHANT_ID: ${{ secrets.BT_MERCHANT_ID }}
-          BRAINTREE_PRIVATE_KEY: ${{ secrets.BT_PRIVATE_KEY }}
-          BRAINTREE_PUBLIC_KEY: ${{ secrets.BT_PUBLIC_KEY }}
-        run: |
-          mvn ${MAVEN_FLAGS} clean install -P${MAVEN_PROFILE}
+  ci:
+    uses: killbill/gh-actions-shared/.github/workflows/ci.yml@main
+    with:
+      test-profile-matrix: '[ "travis", "integration-mysql", "integration-postgresql" ]'
+    secrets:
+      extra-env: '{ "BRAINTREE_ENVIRONMENT": "sandbox", "BRAINTREE_MERCHANT_ID": "${{ secrets.BT_MERCHANT_ID }}", "BRAINTREE_PRIVATE_KEY": "${{ secrets.BT_PRIVATE_KEY }}", "BRAINTREE_PUBLIC_KEY": "${{ secrets.BT_PUBLIC_KEY }}" }'
+
+  integration_test:
+    uses: killbill/gh-actions-shared/.github/workflows/integration_test.yml@main
+    with:
+      plugin_name: 'braintree'
+      integration_tests_goal: 'test:plugins:braintree'
+      ddl_file: 'src/main/resources/ddl.sql'
+      integration_tests_ref: 'refs/heads/work-for-release-0.23.x'
```

Comment:
Maybe we should merge the branch and point all of these CI files to master now.

Reply:
Do you mean merging the `work-for-release-0.23.x` branch of the integration tests repo to `master`? I've already done this and made changes in the other plugins to point to `master`. For example, take a look at the [killbill-stripe/ci.yml](https://github.com/killbill/killbill-stripe-plugin/blob/ec31abcfa6ef3a95a877ce05d6f9db370c17471d/.github/workflows/ci.yml#L21) file.